### PR TITLE
157661420 migrate ci pipeline websockets

### PR DESCRIPTION
**Pull request by echupriyanov**
Replaced Travis CI with GitLab CI

### CODE REVIEW

**Comment by echupriyanov** [comment]
Closed by mistake

**Review by easye — COMMENTED** [review]
Whereas the testing pipeline definitely works, it is unclear to me where the LispWorks values are being specified.  Whereas such things are confidential to Emotiq it would still be a good idea to somehow document how to change their values if we would need to update LispWorks.  Not really sure where such documentation would go.

**Comment by easye** [comment]
I'm a bit nervous about the changes to the gossip included in this pull request, as they seem to indicate to me that things aren't going to go smoothly to get things into `dev`.  As far as I can tell, `157484804-create-websocket-interface-20180529a` is more or less up to date, so I am not sure where these changes are coming from.

**Comment by easye** [comment]
I would like to remove the naming of `$HOME/bin/ros` to stand in for the lisp invocation script, as it is going to confuse the future as to why it is named that way.

**Comment by easye** [comment]
I would like to remove the naming of `$HOME/bin/ros` to stand in for the lisp invocation script, as it is going to confuse the future as to why it is named that way.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    # SYSTEMS: ":cosi-bls-tests :crypto-pairings"
+
+test:macos:ccl:
+  <<: *test_definition
+  tags:
+    - macos
+  variables:
+    LISP: ccl
+
+# .build_test: &build_test_definition
```
Comment (easye) [inline .gitlab-ci.yml:64]: Why is this section commented out?  Will it be used in the future?  If not, it is probably better to delete.

**Comment by echupriyanov** [comment]
I also bit confused, how gossip.lisp ended like this. Lets close this PR for now. I’ll figure out how to bring gossip into proper state, rename lisp-wrapper to something more reasonable, and create new PR

**Comment by easye** [comment]
@echupriyanov `lisp-wrapper` is a fine name.  Just don't symlink that as `ros`…

**Review by easye — COMMENTED** [review]
We want to close this pull request, right?

**Comment by echupriyanov** [comment]
Superseded by later PR

**Pull request closed without merging**